packages/app-extrinsics/src/Extrinsic.tsx

```diff
@@ -73,6 +75,10 @@ class Extrinsic extends React.PureComponent<Props, State> {
   }
 
   nextState (newState: State): void {
+    const isValidLength = (values: Array<RawParam>, index: number) => {
+      return (values[index].type === 'Balance' ? checkValueBitLength(values[index].value as BN) : true);
+    };
```

Comment:
With that however, the bitlength check should be elsewhere, not specifically here. If we do it at the base, we don't need to add the same logic to each component that uses it.

this component should not care about lengths and being valid, what is passed to it should already have the required checks applied.